packages/cli/src/commands/inspect/index.ts

```diff
@@ -197,3 +230,11 @@ function stateString(s: Deployment['readyState']) {
       return chalk.gray('UNKNOWN');
   }
 }
+
+function exitCode(state: Deployment['readyState'], print?: Output['print']) {
+  print?.(`${chalk.cyan('status')}\t${stateString(state)}\n`);
+  if (state === 'ERROR' || state === 'CANCELED') {
+    return 3;
+  }
```

Comment:
Question: why do we want to return a non-0 error code for ERROR/CANCELED? I'm asking because I think some bash scripts could consider that inspecting the deployment failed if they don't receive 0.

Reply:
That is a good question. @EndangeredMassa suggested to [implement a feature request](https://www.notion.so/RFC-split-logs-commands-for-CLI-95800af3532f4b6fb7acf0021247e339?d=74e0ce8458734a51a4a59902886e7e7d&pvs=4#b044f393675f4d459038a010c414cfb9) along with this change.
It'll allow them to detect a failed/cancelled deployment when inspecting logs with the `--wait` flag.